libs/libcommon/src/libcommon/constants.py

```diff
@@ -21,7 +21,7 @@
 PROCESSING_STEP_CONFIG_OPT_IN_OUT_URLS_COUNT_VERSION = 1
```

Comment:
```suggestion
PROCESSING_STEP_CONFIG_OPT_IN_OUT_URLS_COUNT_VERSION = 2
```

Reply:
Otherwise the content will not be updated at config level, thus neither at dataset level, thus in the endpoint we use to check the number of opted-in/opted-out urls.

Reply:
My mistake, thanks!